addons/base_import_module/models/ir_module.py

```diff
@@ -378,10 +378,13 @@ def button_immediate_install_app(self):
                 timeout=5.0,
             )
             resp.raise_for_status()
+            missing_dependencies, is_installation_possible = self._get_missing_dependencies(resp.content)
```

Comment:
so here unavailable module instead of is_installation

rename missing_dependencies in missing_dependencies_description